sztp-agent/pkg/secureagent/daemon_test.go

```diff
@@ -94,29 +110,10 @@ func createTempTestFile(file string, content string, _ bool) {
 	if err != nil {
 		log.Fatal(err)
 	}
-	mydhcpresponse := `lease {
-  interface "eth0";
-  fixed-address 10.127.127.100;
-  filename "grubx64.efi";
-  option subnet-mask 255.255.255.0;
-  option sztp-redirect-urls "http://mymock/test";
-  option dhcp-lease-time 600;
-  option tftp-server-name "w.x.y.z";
-  option bootfile-name "test.cfg";
-  option dhcp-message-type 5;
-  option dhcp-server-identifier 10.127.127.2;
-  renew 1 2022/08/15 19:16:40;
-  rebind 1 2022/08/15 19:20:50;
-  expire 1 2022/08/15 19:22:05;
-}`
-
-	if content != "" {
-		mydhcpresponse = content
-	}
-	_, err2 := f.WriteString(mydhcpresponse)
-
-	if err2 != nil {
-		log.Fatal(err2)
+	defer f.Close()
```

Comment:
small linter error

Reply:
Is this because f.close is not wrapped in a closure function to handle the close error, if any?